cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlApiImpl.java

```diff
@@ -202,38 +202,40 @@ public CruiseControlApiImpl(Vertx vertx, int idleTimeout) {
                                                 json.getString(CC_REST_API_ERROR_KEY)));
                             }
                             JsonObject statusJson = new JsonObject();
-                            String taskStatus = jsonUserTask.getString("Status");
-                            statusJson.put("Status", taskStatus);
-                            // The status could be ACTIVE in which case there will not be a "summary" so we check that we are
-                            // in a state that actually has that key.
-                            if (taskStatus.equals(CruiseControlUserTaskStatus.IN_EXECUTION.toString()) ||
-                                    taskStatus.equals(CruiseControlUserTaskStatus.COMPLETED.toString())) {
-                                // We now need to extract the original response which is in a raw string (not nicely formatted JSON)
-                                statusJson.put("summary", ((JsonObject) Json.decodeValue(jsonUserTask.getString("originalResponse"))).getJsonObject("summary"));
+                            String taskStatusStr = jsonUserTask.getString("Status");
```

Comment:
We use this Status string a few times, maybe worth making it a constant somewhere?